src/Element.js

```diff
@@ -136,6 +137,86 @@ export default class Element extends ParentNode {
         });
         return _array;
     }
+
+    /**
+     * Returns the first element that is a descendant of the element on which it is invoked that matches the
+     * specified group of selectors.
+     * @param {String} query CSS query for selecting element
+     * @return {Element} reference to an Element, or null if element was not found
+     */
+    querySelector(query) {
+        let parser = new CssSelectorParser.CssSelectorParser();
+        let rules = parser.parse(query);
+        var result = null;
```

Comment:
**result** is changed in inner functions (closure) so i used var. Let is not visible bellow isn't it?